code42.json

```diff
@@ -75,26 +93,28 @@
                     "allow_list": true
                 },
                 "start_time": {
-                    "data_type": "numeric",
-                    "order": 1,
-                    "description": "Start of time range, in epoch time (milliseconds)",
-                    "verbose": "If not specified, the default is past 10 days"
+                    "description": "The start time to use in the initial poll in yyyy-MM-dd HH:MM:SS format (defaults to 30 days back)",
+                    "verbose": "Configure this parameter via the `start_date` configuration parameter.",
+                    "data_type": "string",
+                    "required": false,
+                    "order": 1
                 },
                 "end_time": {
-                    "data_type": "numeric",
-                    "order": 2,
-                    "description": "End of time range, in epoch time (milliseconds)",
-                    "verbose": "If not specified, the default is now"
+                    "description": "The end time to use in the initial poll in yyyy-MM-dd HH:MM:SS format (defaults to the current time)",
+                    "verbose": "Configure this parameter via the `end_date` configuration parameter.",
+                    "data_type": "string",
+                    "required": false,
+                    "order": 2
                 },
                 "container_count": {
                     "data_type": "numeric",
                     "order": 3,
-                    "description": "Maximum number of container records to query for."
+                    "description": "Maximum number of alerts to create (only used in Poll Now)"
```

Comment:
Only using these params at all because they always show up in the UI no matter what, and I think if it possible for a user to set a value there, we should honor it...

I have seen some apps completely ignore this param anyway though... Same with `container_id`

It does not hurt, imo, to honor it, since it there... Same with the

Reply:
Also: It only shows up in the UI for `Poll Now`, and there is not really a way to configure it for ongoing polls, so I don't use it for ongoing polls at all (it will get all the alerts it finds)